docs/authentication/saml/authentication-flows.mdx

```diff
@@ -40,16 +43,20 @@ A bad actor could also can intercept the IdP response and replace it with anothe
 Even if the IdP-initiated flow is considered insecure, Clerk tries to mitigate those flaws in order to protect and offer the best possible security for your application and users.
 
 #### Unsolicited `InResponseTo` attribute
+
 In accordance with the [SAML 2.0 profiles specification](https://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf), Clerk ensures that an unsolicited response does not contain an `InResponseTo` attribute.
+
 This prevents bad actors from stealing a response used in an SP-initiated flow and using it in an IdP-initiated flow.
 
 #### Replay detection
```

Comment:
This is a heading under the "Risks of IDP-initiated flow" section. Is "Replay detection" a risk of an IDP-initiated flow?